app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt

```diff
@@ -88,8 +66,15 @@ class FragmentCountriesReports : Fragment() {
 
     private fun bindObservers() {
         viewModel.countryReports.observe(viewLifecycleOwner, Observer {
-            adapter.reports = it
+            adapter.deltas = it
```

Comment:
I don't like `deltas` as a name for  a whole country reports :D

Reply:
Me either xD